src/main/java/de/dennisguse/opentracks/ui/util/ListItemUtils.java

```diff
@@ -160,6 +67,28 @@ private static String getTimeDistance(String totalTime, String totalDistance) {
         return builder.toString();
     }
 
+    public static String getTimeDistanceText(Context context, UnitSystem unitSystem, boolean isRecording, Duration totalTime, Distance totalDistance, int markerCount) {
+        String timeDistanceText;
+        if (isRecording) {
+            timeDistanceText = context.getString(R.string.generic_recording);
+        } else {
+            // Match list_item_time_distance in list_item.xml
+
+            String time = StringUtils.formatElapsedTime(totalTime);
+            // TODO unit system
+            String distance = DistanceFormatter.Builder()
+                    .setUnit(unitSystem)
+                    .build(context).formatDistance(totalDistance);
+
+            timeDistanceText = getTimeDistance(time, distance);
+            if (markerCount > 0) {
+                timeDistanceText += "  ‧";
```

Comment:
?

Reply:
I don't know either, tbh. I just copy-pasted the original code there